conf/messages.en

```diff
@@ -190,7 +190,7 @@ otherEntityType.ctutr.title = What is your Corporation Tax Unique Taxpayer Refer
 otherEntityType.ctutr.h1 = What is your Corporation Tax Unique Taxpayer Reference?
 otherEntityType.ctutr.body = Your Corporation Tax (CT) Unique Taxpayer Reference (UTR) number is 10 digits long.
 otherEntityType.ctutr.hint = For example, 0123456789
-otherEntityType.ctutr.error.required = Enter a Corporation Tax (CT) Unique Taxpayer Reference (UTR)
+otherEntityType.addCtutr.error.required = Enter a Corporation Tax (CT) Unique Taxpayer Reference (UTR)
```

Comment:
why change this key?

Reply:
There was a duplicate. We still have `otherEntityType.ctutr.error.required` and is used when we are asking user does he have a CTUTR. "New" key is used in validation of form when user needs to type in a CTUTR.

Reply:
ok that makes sense, probably worth making the other message keys for this page consistent then e.g. `...addCtutr...` ?

Reply:
Done :)